src/pybind/mgr/nfs/export.py

```diff
@@ -658,6 +663,17 @@ def create_cephfs_export(self,
                              access_type: str,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Dict[str, Any]:
+
+        with open_filesystem(self.fs_client, fs_name) as fs_handle:
+            try:
+                stx = fs_handle.statx(path.encode('utf-8'),
+                                      cephfs.CEPH_STATX_MODE, 0)
+                if not stat.S_ISDIR(stx.get('mode')):
+                    raise NFSException(f"Failed to create export: path {path} "
+                                       f"is not a dir")
+            except cephfs.Error:
+                raise NFSInvalidOperation(f"Failed to create export: "
+                                          f"path {path} is invalid")
```

Comment:
also, use `f""` strings in string segments where there are values to me emitted; if the string segment is just a static string segment, avoid using the `'f'` prefix